pkg/proto/hapi/services/tiller.pb.go

```diff
@@ -129,6 +129,8 @@ type ListReleasesRequest struct {
 	// SortOrder is the ordering directive used for sorting.
 	SortOrder   ListSort_SortOrder          `protobuf:"varint,5,opt,name=sort_order,json=sortOrder,enum=hapi.services.tiller.ListSort_SortOrder" json:"sort_order,omitempty"`
 	StatusCodes []hapi_release1.Status_Code `protobuf:"varint,6,rep,packed,name=status_codes,json=statusCodes,enum=hapi.release.Status_Code" json:"status_codes,omitempty"`
+	// Namespace is the filter to select releases only from a specific namespace.
+	Namespace string `protobuf:"bytes,7,opt,name=namespace" json:"namespace,omitempty"`
```

Comment:
I cannot build this for testing because the `_proto/hapi/services/tiller.proto` file does not have this field in it.

If you haven't already, can you add this line and then `make bootstrap protoc` to regenerate? Once that's done, committing the `.proto` file will fix it.